tmol/database/scoring/ref.py

```diff
@@ -0,0 +1,39 @@
+import yaml
+
+import attr
+import cattr
+
+
+@attr.s(auto_attribs=True, slots=True, frozen=True)
+class RefWeights:
+    ALA: float
+    ARG: float
+    ASN: float
+    ASP: float
+    CYS: float
+    GLN: float
+    GLU: float
+    GLY: float
+    HIS: float
+    ILE: float
+    LEU: float
+    LYS: float
+    MET: float
+    PHE: float
+    PRO: float
+    SER: float
+    THR: float
+    TRP: float
+    TYR: float
+    VAL: float
```

Comment:
I don't think we want to hardcode this.  In theory a user should be able to specify ref weights for other types, as long as base_name matches.  Just loading into a string->value dict should work.